source/module_hamilt_pw/hamilt_ofdft/ml_tools/potential.h

```diff
@@ -0,0 +1,203 @@
+#ifndef POTENTIAL_H
+#define POTENTIAL_H
+
+#include <torch/torch.h>
+#include "./input.h"
+#include "./data.h"
+#include "./kernel.h"
+#include "./grid.h"
+
+class Potential{
```

Comment:
the file name or namespace "Potential" is too common, which will easily cause conflicts, could you think of a more specific class name?